resources/config.test.ci.edn

```diff
@@ -28,28 +28,48 @@
-                                                                             :key-serializer                        "org.apache.kafka.common.serialization.StringSerializer"}}
-                                   :without-producer {:application-id       "test"
-                                                      :bootstrap-servers    "localhost:9092"
-                                                      :stream-threads-count [1 :int]
-                                                      :origin-topic         "topic"
-                                                      :channels             {:channel-1 {:worker-count [10 :int]
-                                                                                         :retry        {:count   [5 :int]
-                                                                                                        :enabled [true :bool]}}}}}
+            :stream-router        {:default                      {:application-id       "test"
+                                                                  :bootstrap-servers    "localhost:9092"
+                                                                  :stream-threads-count [1 :int]
+                                                                  :origin-topic         "topic"
+                                                                  :upgrade-from         "1.1"
+                                                                  :channels             {:channel-1 {:worker-count [10 :int]
+                                                                                                     :retry        {:count   [5 :int]
+                                                                                                                    :enabled [true :bool]}}}
+                                                                  :producer             {:bootstrap-servers                     "localhost:9092"
+                                                                                         :acks                                  "all"
+                                                                                         :retries-config                        5
+                                                                                         :max-in-flight-requests-per-connection 5
+                                                                                         :enable-idempotence                    false
+                                                                                         :value-serializer                      "org.apache.kafka.common.serialization.StringSerializer"
+                                                                                         :key-serializer                        "org.apache.kafka.common.serialization.StringSerializer"}}
+                                   :with-key-val-serdes          {:application-id             "test"
```

Comment:
why have you added a separate `topic-entity` for this. Why not mock config in the test and add or remove values there? This is causing unnecessary repitition in config of this file.